### PR TITLE
Removed width from toolbar

### PR DESCRIPTION
**Pull request by Yash-Ambekar**
Toolbar: More actions #12932 

Fixed the toolbar width issue.

### CODE REVIEW

**Comment by saghul** [comment]
I do t think this is how we want to fix it, it would break small screen setups.

**Comment by Yash-Ambekar** [comment]
> I do t think this is how we want to fix it, it would break small screen setups.

So how are you planning to fix it? with media queries? max width?

**Comment by jluisfa** [comment]
In a quick test, the fix does not seem to affect the small screens.


imagem1

imagem2

**Comment by Yash-Ambekar** [comment]
Even on 512 width it seems fine, for < 512 width it switches to mobile interface.

File: react/features/toolbox/components/web/Toolbox.tsx [hunk 1]
```diff
@@ -366,7 +366,6 @@ const styles = () => {
             margin: 0,
             marginBottom: '8px',
             maxHeight: 'calc(100vh - 100px)',
-            width: '240px'
```
Comment (robertpin) [inline react/features/toolbox/components/web/Toolbox.tsx:369]: Update to `min-width` instead of removing it, please

**Review by robertpin — CHANGES_REQUESTED** [review]

File: react/features/toolbox/components/web/Toolbox.tsx (hunk 1 above)
Comment (Yash-Ambekar) [reply react/features/toolbox/components/web/Toolbox.tsx:369]: Changed it to min-width.

**Comment by Yash-Ambekar** [comment]
I have committed and pushed the changes to yash/styles branch. Please review.
Change: Updated `width: '240px'` to` minWidth: '240px'`.

**Comment by Yash-Ambekar** [comment]
@robertpin can you review the PR please?

**Review by robertpin — APPROVED** [review]

**Pull request merged**